cypress/utils/dashboards/saved_objects_management/commands.js

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+Cypress.Commands.add('selectDataSourceForImport', (dataSourceTitle) => {
```

Comment:
can we reuse existing util to select data source from data source selector?

https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/fc490c3670fae9008867f80f81761e407b7117d0/cypress/utils/commands.js#L605

Reply:
sorry, I tried this one, but I found the selector on import page does not have the input, while the dev tools has this functionality, if reusing, it can break

Reply:
I see, let's use your util for import then. Great finding, let me file an issue on OSD to make "isClearable" prop consistent across all places that consumes this selector in OSD